pytest_ruff.py

```diff
@@ -4,18 +4,18 @@
 from ruff.__main__ import find_ruff_bin
 
 HISTKEY = "ruff/mtimes"
+_mtimes_stash_key = pytest.StashKey[dict[str, float]]()
 
 
 def pytest_addoption(parser):
     group = parser.getgroup("general")
-    group.addoption('--ruff', action='store_true', help="enable checking with ruff")
+    group.addoption("--ruff", action="store_true", help="enable checking with ruff")
 
 
 def pytest_configure(config):
     config.addinivalue_line("markers", "ruff: Tests which run ruff.")
-    if config.option.ruff:
-        if hasattr(config, 'cache'):
-            config._ruffmtimes = config.cache.get(HISTKEY, {})
+    if config.option.ruff and hasattr(config, "cache"):
```

Comment:
Configure could be just disable ruff flag if cache is not defined to avoid checks for cache.

Reply:
No, it should run even without cache. The cache in python side is good to avoid system calls, but ruff has builtin cache too.